great-idea/services.html

```diff
@@ -15,93 +16,136 @@
+        <h3>Digital Design</h3>
+
+        <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor.
+          Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem
+          neque,
+          sed faucibus nibh pellentesque
+          eleifend. Mauris euismod facilisis iaculis.</p>
+
+        <button>Learn More</button>
+      </div>
+
+      <div class="section-box">
+        <h3>UX / UI</h3>
+
+        <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor.
+          Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem
+          neque,
+          sed faucibus nibh pellentesque
+          eleifend. Mauris euismod facilisis iaculis.</p>
+
+        <button>Learn More</button>
+      </div>
+
+      <div class="section-box">
+        <h3>Digital Marketing</h3>
+
+        <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor.
+          Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem
+          neque,
+          sed faucibus nibh pellentesque
+          eleifend. Mauris euismod facilisis iaculis.</p>
+
+        <button>Learn More</button>
+      </div>
+
+      <div class="section-box">
+        <h3>Web Development</h3>
+
+        <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor.
+          Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem
+          neque,
+          sed faucibus nibh pellentesque
+          eleifend. Mauris euismod facilisis iaculis.</p>
+
+        <button>Learn More</button>
+      </div>
+
+      <div class="section-box">
+        <h3>iOS Development</h3>
+
+        <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor.
+          Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem
+          neque,
+          sed faucibus nibh pellentesque
+          eleifend. Mauris euismod facilisis iaculis.</p>
+
+        <button>Learn More</button>
+      </div>
+
+      <div class="section-box">
+        <h3>Android Development</h3>
+
+        <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor.
+          Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem
+          neque,
+          sed faucibus nibh pellentesque
+          eleifend. Mauris euismod facilisis iaculis.</p>
+
+        <button>Learn More</button>
+      </div>
+
+    </section>
+
+    <section class='services-info'>
+      <div class='services-info-details'>
+        <h1>Some Facts About Our Services</h1>
```

Comment:
You should try to not use more then one `<h1>` for page. You could change this as an `<h2>`.